static/js/settings_user_groups.js

```diff
@@ -109,6 +109,24 @@ exports.populate_user_groups = function () {
                 return false;
             }
 
+            for (const element of user_groups_array) {
+                const div_non_editable = $('div.user-group[id="' + element.id + '"]');
+                const div_pill_container = $(
+                    '.pill-container[data-group-pills="' + element.id + '"]',
+                );
+                if (group_data.id !== element.id) {
+                    div_non_editable.find(".name").attr("contenteditable", "false");
+                    div_non_editable.find(".description").attr("contenteditable", "false");
```

Comment:
This seems to be duplicated from what's there inside the `update_membership` function. Could you try reusing that or refactoring to avoid this duplication, if it's really needed?